jobomatic.py

```diff
@@ -12,13 +12,13 @@
 SEARCH_LOCATIONS = [x for x in pandas.read_csv(
     'SearchLocations.csv').to_dict()]
 
-EXLUDE_TERMS = [x for x in pandas.read_csv('JobExclusionTerms.csv').to_dict()]
-
 
 class Listing:
 
     'A definition and properties of Indeed job listings.'
 
+    EXLUDE_TERMS = [x for x in pandas.read_csv('JobExclusionTerms.csv').to_dict()]
```

Comment:
Will this create a new reading action every time class listing is called?

Reply:
No, I think 1 of 2 things will happen:

- "Lazy" initialization, meaning it does the read action only the 1st time it's called
- initialization during construction, meaning it does the read action only during construction

I say "1 of 2 things" because idk which method python is going to do internally.